### PR TITLE
Bug 1115273 - [Stingray][smart-home/smart-system] Implement opening sear...

### PR DESCRIPTION
**Pull request by huchengtw-moz**
...ch bar

### CODE REVIEW

**Comment by try-server-hook** [comment]
[huchengtw-moz John Hu (huchengtw-moz)](https://github.com/huchengtw-moz) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8431b618e722)

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -299,6 +306,11 @@
         this.edit.toggleEditMode();
 
       // Current focus is on a card
```
Comment (dwi2) [inline tv_apps/smart-home/js/home.js]: nit: this comment should move to line 315 or removed

File: tv_apps/smart-home/index.html [hunk 2]
```diff
@@ -130,6 +132,12 @@
           </smart-button>
         </menu-group>
       </section>
+      <section id="search-bar" class="search-bar hidden">
+        <smart-button data-icon="mic" class="search-mic-button fade-out"></smart-button>
+        <div data-icon="search" class="search-input">
+          <span data-l10n-id="search-default-text">search</span>
```
Comment (dwi2) [inline tv_apps/smart-home/index.html]: nit: I think the text `search` between `<span>` and `</span>` is no needed. I did the experiment, l10n will take care of it.

Comment (huchengtw-moz) [inline tv_apps/smart-home/index.html]: Of course, but it can help us to test if we make a typo of l10n id or attribute name.

**Pull request merged**